docs/versioned-plugins/inputs/file-v4.4.0.asciidoc

```diff
@@ -439,7 +444,7 @@ The sincedb record now has a last active timestamp associated with it.
 If no changes are detected in a tracked file in the last N days its sincedb
 tracking record expires and will not be persisted.
 This option helps protect against the inode recycling problem.
-Filebeat has a {filebeat-ref}/inode-reuse-issue.html[FAQ about inode recycling].
+Filebeat has an https://www.elastic.co/guide/en/beats/filebeat/{branch}/inode-reuse-issue.html[FAQ about inode recycling].
```

Comment:
<img width="999" alt="Screen Shot 2022-04-20 at 5 04 36 PM" src="https://user-images.githubusercontent.com/35154725/164323364-e73363cb-63b7-401f-abec-b697f158a869.png">
Note that link is resolving to 8.2 branch rather than `current`.